webapp/channels/src/components/guest_tag/index.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {defineMessages} from 'react-intl';
+import {useSelector} from 'react-redux';
+
+import {GuestTag as PureGuestTag} from '@mattermost/design-system';
```

Comment:
This feels really odd to be importing GuestTag as PureGuestTag inside the index file for the guest_tag component